docs/TravisFileExplained.md

```diff
@@ -41,8 +41,10 @@ php:
 # This section sets up the environment variables for the build.
 env:
  global:
-# This line determines which version of Moodle to test against.
+# This line determines which version branch of Moodle to test against.
   - MOODLE_BRANCH=MOODLE_35_STABLE
+# Optionally, it is possible to specify a different Moodle repo to use:
+# - MOODLE_REPO=git://github.com/username/moodle.git
```

Comment:
Please move these two lines up under line 44 - don't want to confuse them with the following block

Reply:
Addressed in a6a0de7